pkg/collector/container_cgroup_collector.go

```diff
@@ -25,15 +25,17 @@ import (
 
 // updateCgroupMetrics adds container-level cgroup data
 func (c *Collector) updateCgroupMetrics() {
-	klog.V(5).Infof("overall cgroup stats %v", cgroup.SliceHandlerInstance)
-
-	for containerID := range c.ContainersMetrics {
-		cgroup.TryInitStatReaders(containerID)
-		cgroupFSStandardStats := cgroup.GetStandardStat(containerID)
-		for cgroupFSKey, cgroupFSValue := range cgroupFSStandardStats {
-			readVal := cgroupFSValue.(uint64)
-			if _, ok := c.ContainersMetrics[containerID].CgroupFSStats[cgroupFSKey]; ok {
-				c.ContainersMetrics[containerID].CgroupFSStats[cgroupFSKey].SetAggrStat(containerID, readVal)
+	for key := range c.ContainersMetrics {
```

Comment:
I was wondering that:
for each ContainersMetrics has a cgroup `control`, and for each time we do updateCgroupMetrics()
we just get data from `ContainersMetrics.control.stat()` and refresh data there.

current implementation still 
- reload control for each time from disk.(wasted?)
- and a temp memory make as `cGroupMetrics`(wasted?)

Reply:
@SamYuan1990 I changed the logic, now we open the stat only once for the lifetime of the container.